RELEASE_PROCESS.md

```diff
@@ -0,0 +1,26 @@
+## Release Process
+
+Releases are performed via 2 Pull Requests (PR) that implements the release checklist and [GitHub Releases](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).  This promotes consistency and visibility in the process.
+
+### Version Numbers
+
+Version Numbering follows [Semantic Versioning](https://semver.org) standards.  The format is `vMAJOR.MINOR.PATCH`.. ex `v0.1.0`
+
+<img width="753" alt="semver-summary" src="https://user-images.githubusercontent.com/989928/230925438-ac6ac422-6358-4e96-9536-e3f8fc935317.png">
```

Comment:
how do you upload this image and get the URL to be added here?

Reply:
I did a PR using the GitHub File Editor which allows drag and drop of images that will get uploaded.  From there I just reused the link to the image.